Mage.Client/src/main/java/mage/client/dialog/TableWaitingDialog.java

```diff
@@ -89,12 +91,13 @@ public TableWaitingDialog() {
             this.setSize(prefWidth, prefHeight);
         }
 
-        chatPanel.useExtendedView(ChatPanelBasic.VIEW_MODE.NONE);
-        tableSeats.createDefaultColumnsFromModel();
-        TableUtil.setColumnWidthAndOrder(tableSeats, DEFAULT_COLUMS_WIDTH, KEY_TABLE_WAITING_COLUMNS_WIDTH, KEY_TABLE_WAITING_COLUMNS_ORDER);
-        tableSeats.setDefaultRenderer(Icon.class, new CountryCellRenderer());
         setGUISize();
 
+        chatPanel.useExtendedView(ChatPanelBasic.VIEW_MODE.NONE);
+        jTableSeats.createDefaultColumnsFromModel();
+        TableUtil.setColumnWidthAndOrder(jTableSeats, DEFAULT_COLUMNS_WIDTH, KEY_TABLE_WAITING_COLUMNS_WIDTH, KEY_TABLE_WAITING_COLUMNS_ORDER);
```

Comment:
Not clear why this call isn't working - will need to do further debugging but it looks to be called in a similar manner from PlayersChatPanel.java which is working fine